content/contributing/getting-started/using-github-issues.md

```diff
@@ -0,0 +1,38 @@
++++
+title = "Using Github Issues"
+insert_anchor_links = "right"
+[extra]
+weight = 1
++++
+
+Bevy tracks bugs and feature requests using [Github Issues], which is an issue tracker and discussion space. You can find Bevy's issue page [here](https://github.com/bevyengine/bevy/issues), where you can navigate, filter, and view individual issues.
+
+[Github Issues]: https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues
+
+## When to file an issue
+
+Issues are great for formally tracking the progress of bugs and features. Here are some scenarios where you may choose to use issues:
+
+- You want to receive feedback and ideas before writing code.
```

Comment:
should we clarify that "writing code" here means specifically for contributing to Bevy and not just code using Bevy?

Reply:
```suggestion
- You want to receive feedback and ideas before writing code for Bevy.
```